src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt

```diff
@@ -65,15 +62,17 @@ class BuildKmerIndex: CliktCommand(help="Create a kmer index for a HaplotypeGrap
         .default(1)
 
     val hvcfDir by option("--hvcf-dir", help = "Path to directory holding hVCF files. Data will be pulled directly from these files instead of querying TileDB")
-        .default("")
+        .required()//Todo: make this optional by adding .default("")
+
+
 
     override fun run() {
         //either tiledbPath or agcPath must be provided
```

Comment:
this comment could be deleted

Reply:
I added it specifically so we remember to adjust when the direct tileDB connection is up and running.